test/e2e/testsuite/testsuite.go

```diff
@@ -49,6 +50,7 @@ func ParseFlags() {
 	pflag.StringVar(&CRCExecutable, "crc-binary", "", "Path to the CRC executable to be tested")
 	pflag.BoolVar(&cleanupHome, "cleanup-home", false, "Try to remove crc home folder before starting the suite") // TODO: default=true
 	pflag.StringVar(&CRCVersion, "crc-version", defaultCRCVersion(), "Version of CRC to be tested")
+	pflag.StringVar(&CRCMemory, "crc-memory", "9216", "Memory for CRC VM in MiB")
```

Comment:
Maybe better set as "" empty string. If then string is not empty we set the memory otherwise we do nothing / unset the value (to ensure it is the default)

Cause if default value changes in the future we need to remember come here and change it as well